### PR TITLE
fix tuto-03-animated-triangle.md

### PR DESCRIPTION
**Pull request by GraydenH**
Sorry, maybe i was not clear in my last PR. My changes were not working yet.

### CODE REVIEW

**Comment by GraydenH** [comment]
I got the first example working but I was not able to figure out how to pass t to the vertex shader.

**Comment by est31** [comment]
Can you keep this in sync with https://github.com/glium/glium/blob/master/examples/tutorial-03.rs ?

**Comment by est31** [comment]
Also, #1933 does not need a revert. All the PR changed was to replace `poll_events` with `run`. `poll_events` is gone, as is the top level `glutin::Event`. So that PR was an improvement of the situation, which is why I merged it. Further improvements might be needed. For that, one should orient oneself on tutorial-03.rs to keep the two in sync.

Can you remove the "revert" from the title and maybe align the inline sources in the md file with tutorial-03.rs? After that, a squash would be useful.

**Comment by GraydenH** [comment]
I see, The .rs was working but the .md was out of date. I fixed the .md to align with the source code example in a fitting way.

**Comment by est31** [comment]
@GraydenH thanks, is this ready for review?

**Comment by GraydenH** [comment]
yeah, good to go @est31

File: book/tuto-03-animated-triangle.md [hunk 1]
```diff
@@ -113,7 +131,7 @@ let vertex_shader_src = r#"
 You may notice that this is exactly the operation that we've been doing above, except that this time it is done on the GPU side. We have added a variable `t` in our shader, which is declared as a **uniform**. A uniform is a global variable whose value is set when we draw by passing its value to the `draw` function. The easiest way to do so is to use the `uniform!` macro:
 
 ```rust
-target.draw(&vertex_buffer, &indices, &program, &uniform! { t: t },
```
[the remark this answers is not in the document]
Comment (GraydenH) [reply book/tuto-03-animated-triangle.md]: readded the &

**Review by est31 — APPROVED** [review]
Thanks!

**Pull request merged**